scan.conf

```diff
@@ -0,0 +1,793 @@
+##
```

Comment:
Do you think it could be a good idea to only keep the changed values here? It's a bit challenging to find the purpose of this config otherwise because it's hard to spot uncommented lines across commented ones

Reply:
hz, I'm not sure I added all needed options there. so I left it with all commented out possible options to enable it in the future if it is needed